test/integrationtests/voight_kampff/tools.py

```diff
@@ -15,15 +15,161 @@
 """Common tools to use when creating step files for behave tests."""
 
 from threading import Event
+from typing import Any, Callable, List, Tuple
 import time
 
 from mycroft.audio.utils import wait_while_speaking
 from mycroft.messagebus import Message
 
-
 DEFAULT_TIMEOUT = 10
 
 
+class VoightKampffEventMatcher:
```

Comment:
It feels a little confusing to have this called an `EventMatcher`. Could we use `MessageMatcher` instead?